lib/classes/Error.js

```diff
@@ -1,28 +1,49 @@
 'use strict';
+const chalk = require('chalk');
 
-const ServerlessError = class ServerlessError extends Error {
-  constructor(message, messageId) {
+module.exports.SError = class ServerlessError extends Error {
+  constructor(message) {
     super(message);
     this.name = this.constructor.name;
     this.message = message;
-    this.messageId = messageId;
-    Error.captureStackTrace(this, this.constructor.name);
+    Error.captureStackTrace(this, this.constructor);
   }
 };
 
-module.exports = ServerlessError;
-
-module.exports.errorCodes = {
-  UNKNOWN: 1,
-  MISSING_HOMEDIR: 2,
-  MISSING_AWS_CREDS_PROFILE: 3,
-  MISSING_AWS_CREDS: 4,
-  INVALID_PROJ_NAME: 5,
-  ZIP_TOO_BIG: 6,
-  INVALID_PROJECT_SERVERLESS: 7,
-  NO_LAMBDAS_TAGGED_DEPLOYABLE: 8,
-  ACCESS_DENIED: 9,
-  ENV_KEY_NOT_SET: 10,
-  INVALID_RESOURCE_NAME: 11,
-  NOT_IN_SERVERLESS_PROJECT: 12,
+module.exports.logError = (e) => {
+  const consoleLog = (message) => {
+    console.log(message); // eslint-disable-line no-console
+  };
+
+  const errorType = e.name.replace(/([A-Z])/g, ' $1');
+  let line = '';
+  while (line.length < 56 - errorType.length) {
+    line = `${line}-`;
+  }
+
+  consoleLog(chalk.yellow(` ${errorType} ${line}`));
+
+  if (e.name !== 'ServerlessError') consoleLog(' ');
+
+  consoleLog(chalk.yellow(`     ${e.message}`));
```

Comment:
oh the extra space in the first case is because of using string interpolation with the error message. But there's couple of lines of code that handles that issue that you're referencing. However we still need to review error messages to make sure there's no conflicts